libs/sr/db/src/Database/DataManagers/BuildDataManager.ts

```diff
@@ -10,7 +10,7 @@ export interface Build {
   description: string
 
   lightConeId?: string
-  relicIds: Record<RelicSlotKey, string | undefined>
```

Comment:
would prefer if we go by `string | null` for Ids for the future to better adhere to a nullable field in a relational database.

Reply:
https://github.com/users/frzyc/projects/2?pane=issue&itemId=76062503